BST/Node.py

```diff
@@ -0,0 +1,84 @@
+
+import math
+from math import inf
+class Node:
+    # Initializing empty node
+    def __init__(self, value):
+        self.left = None
+        self.right = None
+        self.treasure = value
+    # Function returns the path to a node if exist else return the path to the place in which the node will be added
+    def actual_search(self, treasure):
+        # Returning if node is found
+        if float(treasure) == self.treasure:
+            return ["", self]
```

Comment:
No need to return self, you call the function by the original self

Reply:
You need to return the self of the node we are searching, the original self is the root